AlphaWallet/AppDelegate.swift

```diff
@@ -52,7 +52,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
     func applicationWillResignActive(_ application: UIApplication) {
         protectionCoordinator.applicationWillResignActive()
     }
-    
```

Comment:
Opps!

Reply:
Errorenous whitespace editing? I blame it on AppCode's auto saving :P